rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -42,7 +41,7 @@ class ROCALCOCOIterator(object):
            Epoch size.
     """
 
-    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=120087):
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=8732):
```

Comment:
@SundarRajan28 why is this changing?

Reply:
This argument was added in this iterator to match the ToT coco pipeline example which has 8732 anchors for SSD instead of 120087 anchors. We'll be adding python example for the SSD pipeline in the SSD training PR. For now, I'll remove this argument from this iterator as it is unused